hasbolt.cabal

```diff
@@ -1,5 +1,5 @@
 name:                hasbolt
-version:             0.1.3.3
+version:             0.2.0.0
```

Comment:
I can revert this change before merging if you prefer. I changed the version so it doesn't affect my other builds depending on `0.1.3.3` while building locally.